tests/test_agents/base.py

```diff
@@ -363,16 +357,17 @@ def missing_from_output(  # type: ignore
 
         return missing_strict_strings, missing_round_strings
 
-    @staticmethod
     def __check_missing_strings(
-        missing_strict_strings: List[str], missing_round_strings: List[str], i: int
+        self,
```

Comment:
Do we need `self`? Isn't `__get_agent_name` static?

Reply:
it very well might be, but it's (needlessly) part of a class so need it to reference it using `self` or `cls`

Reply:
alright so name mangling got me good. Let's not use it unless we have a good reason for it :smiling_face_with_tear: